code/game/mecha/mecha_construction_paths.dm

```diff
@@ -466,19 +466,19 @@
 				holder.icon_state = "gygax10"
 		if(8)
 			if(diff==FORWARD)
-				user.visible_message("[user] installs advanced capacitor to the [holder].", "You install advanced capacitor to the [holder].")
+				user.visible_message("[user] installs mech power core to the [holder].", "You install mech power core to the [holder].")
```

Comment:
all of these visible messages need `<span>` s (notice class would probably work for most of these)

Reply:
where would I put the <span>?

Reply:
Like this?
`user.visible_message("<span class='notice'>[user] installs Durand Armor Plates to the [holder].</span>", "<span class='notice'>You install Durand Armor Plates to the [holder].</span>")`

Reply:
Asked for help on discord and have added span class to all notices